Cargo.lock

```diff
@@ -2984,7 +3007,7 @@ dependencies = [
  "futures 0.3.12",
  "futures-timer 3.0.2",
  "lazy_static",
- "libsecp256k1",
+ "libsecp256k1 0.3.5",
```

Comment:
Hmm, what happened here?

Reply:
That is from libp2p.  Upgrading libp2p will require a lot of changes, since there was a lot of API changes from 0.34 -> 0.39.